packages/neuron-wallet/src/controllers/multisig.ts

```diff
@@ -104,27 +115,28 @@ export default class MultisigController {
     }
     try {
       const json = fs.readFileSync(filePaths[0], 'utf-8')
-      let configs: Array<{r: number, m: number, n: number, blake160s: string[], alias: string}> = JSON.parse(json)
-      if (!Array.isArray(configs)) {
-        configs = [configs]
-      }
+      let configOutput: MultisigConfigOutput = JSON.parse(json)
       if (
-        configs.some(config => config.r === undefined
-          || config.m === undefined
-          || config.n === undefined
-          || config.blake160s === undefined
-          || config.r > config.n
-          || config.m > config.n
-          || !config.blake160s.length
-          || config.blake160s.some(v => v.length !== SPEC256_BLAKE160_LEN)
+        !configOutput.multisig_configs ||
```

Comment:
It's kinda complex, maybe we could add a specific function and add test cases to show its rule.